dashboard/app/tree.go

```diff
@@ -112,7 +115,11 @@ type pollResultDone struct {
 }
 
 type bugTreeContext struct {
-	c         context.Context
+	c context.Context
+	// Datastore puts limits on how often a single entity can be accessed by transactions.
+	// And we actually don't always need a consistent view of the DB, we just want to query
+	// a single entity. So, when possible, let's make queries outside of a transaction.
```

Comment:
Is db.Get() a candidate here?
Even if two contexts will work now, we can't guarantee it will remain stable in the future...